src/themes/teams/components/Chat/chatItemStyles.ts

```diff
@@ -1,25 +1,76 @@
 import { ICSSInJSStyle, ComponentSlotStylesInput } from '../../../types'
 import { ChatItemVariables } from './chatItemVariables'
 import { ChatItemProps } from '../../../../components/Chat/ChatItem'
+import { pxToRem } from '../../../../lib'
+import ChatMessage from '../../../../components/Chat/ChatMessage'
+
+const chatMessageClassname = `& .${ChatMessage.className}`
+const chatMessageAuthorClassname = `& .${ChatMessage.className}__author`
+const chatMessageTimestampClassname = `& .${ChatMessage.className}__timestamp`
 
 const chatItemStyles: ComponentSlotStylesInput<ChatItemProps, ChatItemVariables> = {
-  root: ({ props: p, variables: v }): ICSSInJSStyle => ({
-    position: 'relative',
-    marginTop: v.margin,
-    marginBottom: v.margin,
-  }),
+  root: ({ props: p, variables: v }): ICSSInJSStyle => {
+    const { grouped } = p
+    return {
+      position: 'relative',
+      ...((!grouped || grouped === 'start') && { marginTop: pxToRem(16) }),
+      ...(grouped &&
+        grouped !== 'start' && {
+          marginTop: pxToRem(2),
+          [chatMessageAuthorClassname]: {
+            display: 'none',
+          },
+          [chatMessageTimestampClassname]: {
+            display: 'none',
+          },
+        }),
+      marginBottom: 0,
+    }
+  },
 
-  gutter: ({ props: p, variables: v }): ICSSInJSStyle => ({
-    position: 'absolute',
-    marginTop: v.gutterMargin,
-    [p.gutterPosition === 'end' ? 'right' : 'left']: 0,
-  }),
+  gutter: ({ props: p, variables: v }): ICSSInJSStyle => {
+    const { grouped } = p
+    return {
+      position: 'absolute',
+      marginTop: v.gutterMargin,
+      [p.gutterPosition === 'end' ? 'right' : 'left']: 0,
+      ...(grouped &&
+        grouped !== 'start' && {
+          visibility: 'hidden',
```

Comment:
we have `display: 'none'` on the author and timestamp and `visibility: 'hidden'` here, any reason we cannot keep consistency? I'd expect `display: 'none'` to work here as well